packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.stories.ts

```diff
@@ -16,6 +16,7 @@ const createStory = storyWithProps(
     link: 'http://localhost:3030/connect_another_device',
     desktopLink: 'https://firefox.com',
     onDesktopOrTabletDevice: true,
+    productName: 'Firefox',
```

Comment:
Just curious with everything being .ftl changes, why was this needed?

Reply:
This was a fix so this story would render (missing prop was causing it to error)